docker/cassandra-operator/Makefile

```diff
@@ -1,8 +1,15 @@
 
+# Figure out the path to the root of the repository
+rootdir = $(shell git rev-parse --show-toplevel)
+# Obtain operator version from Git
+git_revision = $(shell git describe --tags --always --dirty)
+
 .PHONY: cassandra-operator
 cassandra-operator:
-	docker build \
+	cd $(rootdir) && docker build \
+		--build-arg version=$(git_revision) \
 		-t $(DOCKER_REPO)/cassandra-operator \
```

Comment:
so I guess we should remove `/` here after `DOCKER_REPO`

Reply:
Thanks, fixed.

For the record, this is necessary following https://github.com/instaclustr/cassandra-operator/pull/207#issuecomment-518395482.